tests/nnx/transforms_test.py

```diff
@@ -55,6 +55,10 @@ def __getattr__(self, key): ...
 
 
 class TestJIT(absltest.TestCase):
+  @classmethod
+  def setUpClass(cls):
+    jax.config.update('jax_num_cpu_devices', 4)
```

Comment:
I'm not totally sure how safe this line can be in the context whether this test class could be called on a large infrastructure with multiple devices.

Reply:
By default, they are:

Virtual Machine | Processor (CPU) | Memory (RAM) | Storage (SSD) | Architecture | Workflow label
-- | -- | -- | -- | -- | --
Linux | 4 | 16 GB | 14 GB | x64 | ubuntu-latest, ubuntu-24.04, ubuntu-22.04


https://docs.github.com/en/actions/how-tos/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories